views/albums.hbs

```diff
@@ -0,0 +1,6 @@
+<div>
+  {{#each allAlbums}}
+    <img src={{this.images.0.url}} alt="" width="100px" height="100px" />
+  {{/each}}
+  <a href="/albums/someArtistIdGoesHere">View Albums</a>
```

Comment:
This `<a> tag` should be inside the `#each allAlbums`, you needed to receive the `id` of the album in your URL, to gather it on your `req.params` and make the call to the API. 
```
{{#each allAlbums}}
    <img src={{this.images.0.url}} alt="" width="100px" height="100px" />
    <a href="/viewtracks/{{this.id}}">View Tracks</a>
{{/each}}  
```